catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/SearchController.java

```diff
@@ -71,8 +71,7 @@
     /**
      * Create a new SearchController
      *
-     * @param framework
-     *            - CatalogFramework that will be handling the actual queries
+     * @param framework     - CatalogFramework that will be handling the actual queries
```

Comment:
Remove all the `-` in the `@param` descriptions and use `{@link CatalogFramework}` here.